### PR TITLE
Continuous Runner

### PR DESCRIPTION
**Pull request by dzajkowski**

### CODE REVIEW

**Comment by lukasz-golebiewski** [comment]
I'm not sure how this is supposed to work with drone

File: templater/runner/src/test/scala/coop/rchain/perf/DeployProposeSimulation.scala [hunk 1]
```diff
@@ -42,11 +46,13 @@ class DeployProposeSimulation extends Simulation {
   val protocol = RNodeProtocol(host, port)
 
   val scn = scenario("DeployProposeSimulation")
-    .repeat(2) {
-      exec(deploy(contract))
-        .pause(1)
-        .exec(propose())
-        .pause(1)
+    .foreach(List(contract), "contract") {
```
Comment (lukasz-golebiewski) [inline templater/runner/src/test/scala/coop/rchain/perf/DeployProposeSimulation.scala]: Why did you do this unrelated change?

Comment (dzajkowski) [reply templater/runner/src/test/scala/coop/rchain/perf/DeployProposeSimulation.scala]: sadly it's not unrelated. the way a param can be passed to a scenario is via EL expressions. it's stupid, I don't like it but that's how the whole gatling framework is organised.
this change is to align with the way params are being passed.
